tests/neo.UnitTests/SmartContract/Native/Tokens/UT_NeoToken.cs

```diff
@@ -389,6 +389,8 @@ public void TestGetValidators2()
         {
             var snapshot = Blockchain.Singleton.GetSnapshot();
             var result = NativeContract.NEO.GetValidators(snapshot);
+            result[0] = result[1]; // tampering the data
+            result = NativeContract.NEO.GetValidators(snapshot);
```

Comment:
The UT could ensure that the references are different, right?

Reply:
Yes